celltk/preprocess_operation.py

```diff
@@ -133,6 +155,7 @@ def flatfield_references(img, ff_paths=['Pos0/img00.tif', 'Pos1/img01.tif'], exp
     for path in store:
         ff_store.append(imread(path))
     ff = np.median(np.dstack(ff_store), axis=2)
+    ff = np.squeeze(ff) # added by Katie to fix bug in new version of numpy
```

Comment:
Which version of numpy are you using @kbodner? Is it the same one @braysia is using? Just want to make sure we aren't introducing incompatibilities in the dependencies.

Reply:
v1.13.3. I believe these issues started after Jerry performed an update on numpy.

Reply:
Ah okay, so we are all on the same version then. Sounds good, approved from me.

Reply:
I'm using v1.14.6 and I couldn't reproduce the issue. I guess this suggests the way you pass ff_paths might not be correct - I'm happy to see the input file if you have one.

Reply:
@prismofeverything
All the changes in existing functions should be discarded as those errors seems to be expected behaviors.
I just want to accept the addition of functions. What do you think the easy way to go forward? Re-send pull requests?

Reply:
done. I deleted the extra lines I added to fix perceived numpy bugs

Reply:
@kbodner Awesome! Once you push to this branch we should be ready to merge then.